lib/assets/javascripts/markdown-it-diaspora-linkify.js

```diff
@@ -2,69 +2,11 @@
-},{}],4:[function(require,module,exports){
 // Replace link-like texts with link nodes.
 //
 'use strict';
 
-var urlRegex = require('url-regex');
-var LINK_SCAN_RE = /www|@|\:\/\//;
+var urlRegex = /(?:^|\s)(?:(?:https?|ftp):\/\/|mailto:|xmpp:|www[^.\s]*\.)\S+/gi;
```

Comment:
I am not sure if this is really much worse than the regex we had before: https://www.debuggex.com/r/4eJ2IFhc7VXnEOKu

I could add a mandatory `@` for `mailto:` and `xmpp:` in the regex. (`mailto:\S+@\S+`)
Which protocols does twitter-text-rb support?